rs/tests/driver/src/sns_client.rs

```diff
@@ -452,11 +452,6 @@ async fn deploy_new_sns_via_proposal(
     let runtime = runtime_from_url(nns_node.get_public_url(), nns_node.effective_canister_id());
     let nns_agent = nns_node.build_canister_agent().await;
 
-    // Sanity check that params is valid
-    SnsInitPayload::try_from(create_service_nervous_system_proposal.clone()).expect(
```

Comment:
This happens in the canister already, and removing this allows us to stop depending on ic_nns_governance types